content/english/java-tictactoe/activity-5.md

```diff
@@ -3,4 +3,68 @@ title: "5. (Optional) AI Move"
 description: ""
 date: 2020-07-23T00:00:00Z
 weight: 6
----
\ No newline at end of file
+hidden: false
+---
+
```

Comment:
Emphasize that this is a bonus exercise, only for those ready for the challenge.